### ISSUE TITLE
SSL not working on linux ubuntu 20.20 

### ISSUE THREAD

**Issue by hananloser** [cpriego/valet-linux#306]
![image](https://user-images.githubusercontent.com/23658759/96923676-3806c980-14e4-11eb-9674-09f45349f80e.png)

- iam try change owner file still not working i have no idea what should i do 
![image](https://user-images.githubusercontent.com/23658759/96923849-6c7a8580-14e4-11eb-972a-615a3d7f7af9.png)

![image](https://user-images.githubusercontent.com/23658759/96924033-a8ade600-14e4-11eb-8569-a07e62cccf03.png)

**Comment by josegus** [comment]
Same here, use to work with ubuntu 19. I'm using ubuntu 20.04 and valet 2.2.6, I've tried deleting the ssl and generated again, but still not working.
![image](https://user-images.githubusercontent.com/13947849/97196049-6570a300-1782-11eb-955d-87a9eb9acd3c.png)

**Comment by dougblackjr** [comment]
Also getting this issue. Certificate is invalid, even when trying to import directly in the browser.

**Comment by josegus** [comment]
@hananloser @dougblackjr could you try the steps suggested [on this thread](https://github.com/cpriego/valet-linux/issues/292#issuecomment-716652529)? I've followed but still didn't work for me

**Comment by dougblackjr** [comment]
@josegus Thank you for that. `sudo update-ca-certificates --fresh` solved it for me

**Comment by josegus** [comment]
@dougblackjr help me please. did you import the certifica to chrome? which file I have to import?

**Comment by dougblackjr** [comment]
@josegus I may have spoken too soon. It secured one site but nothing else will secure. Not sure why.

**Comment by josegus** [comment]
share and aks if your teammates have the same issue, I work with a team of three people and none could make it work

**Comment by ivo-pontes** [comment]
Hi, guys, same problem here. Actually, it works until I create a new domain, then valet breaks.

**Comment by ivo-pontes** [comment]
Aparentilly, we cannot use reserved TLD such as .dev and .app, I've seen in [issue 280](https://github.com/cpriego/valet-linux/issues/280). So, I've tested with domain _.orc_ and my problem was solved. I only have one question, why  do the documentation uses _.app_ as an example if it's a reserved TLD(Top Level Domain)?

**Comment by josegus** [comment]
> Aparentilly, we cannot use reserved TLD such as **.dev** and **.app**, I've seen in [issue 280](https://github.com/cpriego/valet-linux/issues/280). So, I've tested with domain **.orc** and my problem was solved. I only have one question, why do the documentation use **.app** as an example if it's a reserved TLD(Top Level Domain)?

I knew that .app and .dev are reserved tld, but I think we are using .test

**Comment by malicucen** [comment]
I've the same issue. I tried the solution suggestions but none of them worked for me.

**Comment by wilmanbarrios** [comment]
I'm having the same issue with the version v2.2.17.

**Comment by jmarcher** [comment]
I can not recreate this issue, I mean we can not get rid of the warning on the browser, we are self-signing the certificates and it is natural that they are treated as "insecure".

**Comment by stickeerehan** [comment]
Issue is not limited to Ubuntu 20.20. 

On Ubuntu 20.04.1 LTS, upgraded valet to 2.2.17 and my previously secured sites now display the same NET::ERR_CERT_AUTHORITY_INVALID error on Chrome.

Followed the instructions:
```
sudo cp ~/.valet/CA/LaravelValetCASelfSigned.pem /usr/local/share/ca-certificates/LaravelValetCASelfSigned.crt
sudo update-ca-certificates --fresh
```

No errors, and can see `Replacing debian:LaravelValetCASelfSigned.pem` in the log, however site is still showing error.

Also tried `valet secure` to create a fresh certificate. Logs show it was successful:

```
Restarting php7.4-fpm...
Restarting nginx...
The [site.test] site has been secured with a fresh TLS certificate
```
 However, that did not fix it, and running `valet secured` returns 'is not secured.'

**Comment by gsmeira** [comment]
I'm facing the same problem after upgrade to version 2.2.17 on Ubuntu 20.04.1 LTS.

**Comment by dougblackjr** [comment]
Ok, I _think_ I found a workaround. If I'm reading this right, it looks like the CA isn't being imported properly.

Found a possible solution here: [https://superuser.com/a/719047](https://superuser.com/a/719047)

Ran:
```
sudo cp ~/.valet/CA/LaravelValetCASelfSigned.pem /usr/local/share/ca-certificates/LaravelValetCASelfSigned.crt
sudo update-ca-certificates
valet restart
```

And then cleared browser cache (Chrome was caching the SSL certs) and restarted the browser. My sites seem to be secured now. 

I know this doesn't help address the issue in the code. Still looking in to it there.

**Comment by slakbal** [comment]
Can get valet going anymore since I updated with this problem. Anyway has some solution?

**Comment by slakbal** [comment]
Also have the same problem and tried everything above. Any solution from anyone?

**Comment by srosano** [comment]
Did you edit the ~/.config/valet/Nginx/app-name.test file?

**Comment by ctsstc** [comment]
I tried going into Chrome: `chrome://settings/certificates?search=certificate` but I'm having no luck playing around in the `~/.valet/` `CA` or `Certificates` just get a "The Private Key for this Client Certificate is missing or invalid"

**Comment by simple-hacker** [comment]
Sorry to do a little hijack.  I'm having trouble securing sites.  No matter what directories I've parked, and what folders I am trying to secure, whenever I run `valet secure` only the "root" site is secured `The [root.test] site has been secured with a fresh TLS certificate.`  Can anybody help please?

**Comment by vanthao03596** [comment]
> Sorry to do a little hijack. I'm having trouble securing sites. No matter what directories I've parked, and what folders I am trying to secure, whenever I run `valet secure` only the "root" site is secured `The [root.test] site has been secured with a fresh TLS certificate.` Can anybody help please?

Same issue here

**Comment by dougblackjr** [comment]
@jmarcher Can you clarify why this was closed? It appears a number of us are still having this issue.

**Comment by josegus** [comment]
It is not closed, this issue has been present since the end of 2018

**Comment by jmarcher** [comment]
Here are a few "solutions" but the thing is I can not see this as a problem, valet can only generate self signed certificates, all browsers will see this as an insecure certificate. There is no way (or no way I know) how to produce a secure and valid certificate trusted by the browsers and I do not see this as a vital requirement for local development.

**Comment by dougblackjr** [comment]
So I just had to update Composer and Valet because everything got disconnected, and this SSL issue is back. Can anyone give an update on what is being done around this?

**Comment by Floctopus** [comment]
@dougblackjr I know this is already a while back, but im running into this issue as well now. Have you managed to find a more permanent fix for this already by any chance?

**Comment by dougblackjr** [comment]
@Floctopus I was able to get this back up and working following the steps I left in [this comment](https://github.com/cpriego/valet-linux/issues/306#issuecomment-729664012). Not ideal, and doesn't work for everyone (it sounds like).  No real permanent fix at this point, and I haven't had a chance to fork this yet.

**Comment by mpemburn** [comment]
Glad to see this today.  Since I last visited here, I have created a Ubuntu 21 virtual machine on my Mac to use as an alternate dev environment (for the occasions when I want to work on different branches of a project at the same time).  I found a fork of Valet for Linux—Valet Linux+—that was easy to install, and works perfectly.  Here's where to find it:

https://valetlinux.plus/

The evidence:
![image](https://user-images.githubusercontent.com/461989/154844764-ae440a52-bfa8-40f8-8198-6580b1658601.png)
